crates/cxx-qt-gen/src/generator/rust/cxxqttype.rs

```diff
@@ -8,57 +8,62 @@ use crate::{
     naming::TypeNames,
 };
 use quote::quote;
-use syn::Result;
+use syn::{Ident, Result};
 
 use super::fragment::RustFragmentPair;
 
 pub fn generate(
     qobject_ident: &QObjectNames,
     type_names: &TypeNames,
+    module_ident: &Ident,
```

Comment:
once #1060 lands or if this lands first, either branch will need updating to the ` qobject_names.name.module_ident()` route, cc @BenFordTytherington